src/components/likeButton/LikeButton.test.tsx

```diff
@@ -0,0 +1,16 @@
+import 'react-native';
+import React from 'react';
+import {test, describe, expect} from '@jest/globals';
+import LikeButton from './LikeButton';
+import {render} from '@testing-library/react-native';
+
+describe('LikeButton tests', () => {
+  test('Renders the LikeButton correctly', () => {
+    const tree = render(<LikeButton />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  test('LikeButton render works correctly', () => {
+    render(<LikeButton />);
+  });
```

Comment:
It'd be great to also add other tests for like button. I suppose it should change styling, depending on if it's "liked" or not.

Reply:
At first, I added animation for this button, but then I canceled it. But you are right that if there are different behaviors, then it is worth describing them in the tests.